egs/swbd/s5c/local/nnet3/multi_condition/run_ivector_common.sh

```diff
@@ -0,0 +1,148 @@
+#!/bin/bash 
+#set -e
+# This script is based on local/nnet3/run_ivector_common.sh.
+# It reverberates the original data with simulated room impulse responses
+
+. cmd.sh
+
+stage=1
+num_data_reps=1  # number of reverberated copies of data to generate
+input_data_dir=train_nodup
+ivector_dir=exp/nnet3_rvb
+speed_perturb=true
+
+set -e
+. cmd.sh
+. ./path.sh
+. ./utils/parse_options.sh
+
+mkdir -p $ivector_dir
+
+if [ "$speed_perturb" == "true" ]; then
+  # perturbed data preparation
+  if [ $stage -le 1 ] && [ ! -d data/${input_data_dir}_sp ]; then
+    #Although the nnet will be trained by high resolution data, we still have to perturbe the normal data to get the alignment
+    # _sp stands for speed-perturbed
+
+    for datadir in ${input_data_dir}; do
+      utils/perturb_data_dir_speed.sh 0.9 data/${datadir} data/temp1
+      utils/perturb_data_dir_speed.sh 1.1 data/${datadir} data/temp2
+      utils/combine_data.sh data/${datadir}_tmp data/temp1 data/temp2
+      utils/validate_data_dir.sh --no-feats data/${datadir}_tmp
+      rm -r data/temp1 data/temp2
+
+      mfccdir=mfcc_perturbed
+      steps/make_mfcc.sh --cmd "$train_cmd" --nj 50 \
+        data/${datadir}_tmp exp/make_mfcc/${datadir}_tmp $mfccdir || exit 1;
+      steps/compute_cmvn_stats.sh data/${datadir}_tmp exp/make_mfcc/${datadir}_tmp $mfccdir || exit 1;
+      utils/fix_data_dir.sh data/${datadir}_tmp
+
+      utils/copy_data_dir.sh --spk-prefix sp1.0- --utt-prefix sp1.0- data/${datadir} data/temp0
+      utils/combine_data.sh data/${datadir}_sp data/${datadir}_tmp data/temp0
+      utils/fix_data_dir.sh data/${datadir}_sp
+      rm -r data/temp0 data/${datadir}_tmp
+    done
+  fi
+
+
+  if [ $stage -le 2 ]; then
+    #obtain the alignment of the perturbed data
+    steps/align_fmllr.sh --nj 100 --cmd "$train_cmd" \
+      data/${input_data_dir}_sp data/lang_nosp exp/tri4 exp/tri4_ali_nodup_sp || exit 1
+  fi
+
+  clean_data_dir=${input_data_dir}_sp
+else
+  clean_data_dir=${input_data_dir}
+fi
+
+
+if [ $stage -le 3 ]; then
+  if [ ! -d "simulated_rirs_8k" ]; then
+    # Download the simulated RIR package with 8k sampling rate
+    wget --no-check-certificate http://www.openslr.org/resources/26/sim_rir_8k.zip
+    unzip sim_rir_8k.zip
+  fi
+
+  # corrupt the data to generate reverberated data 
+  # this script modifies wav.scp to include the reverberation commands, the real computation will be done at the feature extraction
+  # if --include-original-data is true, the original data will be mixed with its reverberated copies
+  python steps/data/reverberate_data_dir.py \
+    --prefix "rev" \
+    --rir-set-parameters "0.3, simulated_rirs_8k/smallroom/rir_list" \
```

Comment:
what happens to the other 0.1 probability mass ? could you add a comment here describing how these weights are used ?